templates/index.html

```diff
@@ -1,16 +1,17 @@
 {% extends "base_index.html" %}
 
 {% block takeover_content %}
-  <section id="takeover" class="js-takeover">
-    {% include "takeovers/_2004-takeover.html" %}
+  <section id="takeover" class="js-takeover">   
```

Comment:
We should keep one takeover in here so that something is displayer to users without JS